lib/merit/models/active_record/merit/score.rb

```diff
@@ -7,7 +7,9 @@ class Score < ActiveRecord::Base
              class_name: 'Merit::Score::Point'
 
     def points
-      score_points.group(:score_id).sum(:num_points).values.first || 0
+      score_points.select("COALESCE(SUM(num_points), 0) AS num_points").
```

Comment:
Do we need the `COALESCE` call, given `num_points` has a `default: 0` option set? I'd rather convert it to `null: false`.

Anyhow, do you prefer the version with SQL? The Ruby version seems more straightforward to me. If it has significant performance gains, we'll need to back them up with measurements. :)